src/pep/stringpairlist.rs

```diff
@@ -0,0 +1,533 @@
+//! A reimplementation of the engine's stringpairlist module in Rust.
```

Comment:
There seems to be some issue when this is imported to C or when created, like initially we have a stringpair_list with a pair of empty strings ("", ""). This seems to be causing some havoc.